src/contexts/Note/NoteContext.jsx

```diff
@@ -27,8 +28,14 @@ const NoteProvider = ({ children }) => {
   };
 
   const [note, setNote] = useState(initialData);
+  const { isPinned } = note;
+  const [showLabel, setShowLabel] = useState(false);
+  const [showColor, setShowColor] = useState(false);
+  const [showPriority, setShowPriority] = useState(false);
```

Comment:
Since the appearance of these 3 sections depend on one another,you can use an object in this format
```
const [activeSection,setActiveSection]=useState({
showLabel:false,
showColor:false,
showPriority:false
})
```
And then whenever a change in state occurs you can use the setActiveSection state update function

Reply:
Yes, that's a better approach. Will refactor it.